Documentation/cmdref/cilium-operator.md

```diff
@@ -49,6 +49,8 @@ cilium-operator [flags]
       --operator-api-serve-addr string          Address to serve API requests (default "localhost:9234")
       --operator-prometheus-serve-addr string   Address to serve Prometheus metrics (default ":6942")
       --parallel-alloc-workers int              Maximum number of parallel IPAM workers (default 50)
+      --subnets-ids strings                     Subnets IDs (separated by commas)
+      --subnets-tags stringToString             Subnets tags in the form of k1=v1,k2=v2 (multiple k/v pairs can also be passed by repeating the CLI flag (default [])
```

Comment:
What about naming this `subnet-tags-filter` ? We already have `eni-tags` which is used to be passed when creating AWS ENIs and don't want to have some confusions that the operator will create subnets as well.